File: test/langs/sample.cs
namespace Payments
{
    public class Ledger
    {
        private long balance;

        public Ledger()
        {
            balance = 0;
        }

        public void Deposit(long amount)
        {
            balance += amount;
        }
    }

    interface IAccount
    {
        long Balance();
    }
}

File: test/langs/sample.go
package payments

import "fmt"

type Ledger struct {
	balance int
}

func NewLedger() *Ledger {
	return &Ledger{balance: 0}
}

func (l *Ledger) Deposit(amount int) {
	l.balance += amount
	fmt.Println("deposited", amount)
}
